package.json

```diff
@@ -154,8 +156,9 @@
     "@uniswap/v3-core": "1.0.0",
     "@uniswap/v3-periphery": "^1.1.1",
     "@walletconnect/ethereum-provider": "^1.7.1",
-    "@web3-react/metamask": "8.0.24-beta.0",
-    "@web3-react/walletconnect": "8.0.31-beta.0",
+    "@web3-react/metamask": "8.0.27-beta.0",
+    "@web3-react/network": "8.0.27-beta.0",
+    "@web3-react/walletconnect": "8.0.35-beta.0",
```

Comment:
I think these should be in `dependencies` not `devDependencies`

Reply:
yes ^